pyccel/stdlib/cuda_ndarrays/cuda_ndarrays.cu

```diff
@@ -132,6 +132,19 @@ t_ndarray   cuda_array_create(int32_t nd, int64_t *shape,
     return (arr);
 }
 
+void cupy_ravel(t_ndarray *dest, t_ndarray2 src, enum e_memory_locations location)
+{
+    *dest = src;
+    void (*fun_ptr_arr[])(void*, size_t) = {managed_memory, host_memory, device_memory};
+
+    dest->nd = 1;
+    (*fun_ptr_arr[location])(&(dest->shape), sizeof(int64_t));
+    (*fun_ptr_arr[location])(&(dest->strides), sizeof(int64_t));
+    *(dest->shape) = src.lenght;
```

Comment:
Type `lenght` -> `length`. Does this mean that this function isn't tested or that there is a typo throughout the code base?

Reply:
Thats a typo from my side, i can't really test my changes locally for now since i don't have access to a machine with a Nvidia GPU for now, so i mostly only verify the printing side and count on @bauom to verify the compilation.

Reply:
Ah. Are there no compilation tests then? You don't need a GPU to compile, just to execute

Reply:
Pretty much i didn't test compilation, i'll try to install nvcc this time and see how it goes.

Reply:
Fixed in [27c4250](https://github.com/pyccel/pyccel/pull/1222/commits/27c42509cd49ed0d43dc62a1788e1bec1b4d5c12).